components/Navbar/NavbarComponentsBuilder.tsx

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import prisma, { Prisma } from "@/lib/prisma";
+import { NavbarItemBuilder } from "./NavbarItem";
+import { NavbarDropdownBuilder } from "./NavbarDropdownBuilder";
+import { ImageComponent } from "../Shared/ImageComponent";
+import { getAssetPath } from "@/lib/getAssetInfo";
+import { NavbarComponents } from "./NavbarComponentsComponent";
+
+type NavbarComponentsBuilderProps = Prisma.NavbarGetPayload<{}>;
+type NavbarComponentBuilderProps = Prisma.NavbarComponentsGetPayload<{}>;
+
+export async function NavbarComponentsBuilder({
+  dateUpdated,
+  logo,
+  id,
+  userUpdated,
```

Comment:
Use e.g. `...navbar` instead of unpacking each variable. If you don't want `logo` to be in this variable, unpack it separately. The ellipsis (`...`) only takes the variables that aren't unpacked.